File: gracie.lee/src/main/java/homework/chapter_6/homework3/Product.java
package homework.chapter_6.homework3;

import jakarta.persistence.*;

@Entity(name = "Hw3Product")
@Table(name = "product_hw3")
public class Product {

    @Id
    @GeneratedValue
    @Column(name = "product_id")
    private Long id;

    private String name;

    protected Product() {
    }

    public Long getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: hyun.k/hyun.k/src/main/kotlin/chapter6/homework3/Product.kt
package chapter6.homework3

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity(name = "Hw3ProductKt")
@Table(name = "product_hw3_kt")
data class Product(
    @Id
    @GeneratedValue
    @Column(name = "product_id")
    val id: Long? = null,

    var name: String? = null
)